libs/blocks/reading-time/reading-time.css

```diff
@@ -0,0 +1,11 @@
+main > .section > .content.reading-time > span {
+  color: var(--color-gray-600);
```

Comment:
Nit: 
Do we need this scoped so strict? I could see it being as simple as `.reading-time > span {}`